src/Models/Account.js

```diff
@@ -246,6 +246,19 @@ class Account {
         return responseData;
     }
 
+    async reportSSOtoken(ssoDomain, tokenData, retire = false) {
+        const body = new FormData();
+
+        body.append('token', tokenData.token);
+        body.append('delete', retire ? 1 : 0);
+
+        fetch(this.config.API.reportSSOtoken(ssoDomain), {
+            method: 'POST',
+            body: body,
```

Comment:
`body: body` can be replaced with just `body` [(property shorthand notation)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015)